### PR TITLE
Add module about extended cron lines

### PR DESCRIPTION
**Pull request by adamlazik1**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.4 or older.

### CODE REVIEW

File: guides/common/modules/con_using-extended-cron-lines.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+You can use `#` to specify a concrete week day in a month::
+For example:
++
+* `0 0 * * mon#1` specifies first Monday of the month
+* `0 0 * * fri#3,fri#4` specifies 3rd and 4th Fridays of the month
+* `0 7 * * fri#-1` specifies the last Friday of the month at 07:00
+* `0 7 * * fri#L` also specifies the last Friday of the month at 07:00
+* `0 23 * * mon#2,tue` specifies the 2nd Monday of the month and every Tuesday, at 23:00
+
+You can use `%` to specify every X-th day  of the month::
```
Comment (maximiliankolb) [inline guides/common/modules/con_using-extended-cron-lines.adoc:18]:
```suggestion
You can use `%` to specify every n-th day  of the month::
```

**Review by maximiliankolb — COMMENTED** [review]
I did not check all examples, but overall, looks good! Maybe add a link to https://crontab.guru/ or similar, at least for foreman-el,foreman-deb, and katello?

File: guides/common/modules/con_using-extended-cron-lines.adoc (hunk 1 above)
Comment (adamlazik1) [reply guides/common/modules/con_using-extended-cron-lines.adoc:18]: Applied.

**Comment by adamlazik1** [comment]
> I did not check all examples, but overall, looks good! Maybe add a link to https://crontab.guru/ or similar, at least for foreman-el,foreman-deb, and katello?

Thanks! I added the link for katello and foreman builds.

The examples were verified by SME, and were taken mostly from https://github.com/floraison/fugit#fugitcron so I believe they should be good.

**Comment by adamlazik1** [comment]
Thanks for the feedback @maximiliankolb! Applied and merging.

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   4c40d9ac1..7cc353b85  3.7 -> 3.7

**Pull request merged**